[Process/Feature] Save previous version to correctly show the actual version when a bundle has been rolled back

NielsSmits1 opened the pull request:
## Why
I was adding react-native-ota-update to a project, when I found out the version is not set back in any way when a bundle is rolled back. I found it strange, since the bundle itsself does reset. This might end up being a hazard when a user is reporting bugs from an OTA version, he doesn't even have.

## How
Every time the version is set, I check if another version was already saved at that point. If it was, I was save that version as the previous version.

## Related Issue
I did not create an issue for this;

NielsSmits1 commented:
Hi @vantuan88291,

I really appreciate the package that you have made. Especially the expo plugin is a livesaver. I hope you can check this soon.

vantuan88291 commented:
Hi, thanks for your PR, you mean if we rollback to previous bundle, now it show wrong version, right?

NielsSmits1 commented:
Hi @vantuan88291 ,

On your first comment:

> Hi, thanks for your PR, you mean if we rollback to previous bundle, now it show wrong version, right?

That is indeed the use case I added it for.

In my last commit I removed the redundant synchronize. I moved the previous_version adjustment into the check if the new version is not empty. Since I believe the previous version should only be changed when the actual version changes.

vantuan88291 commented:
I think it good, will deploy soon. Thanks
